### DISCUSSION TITLE
Custom properties not showing up in settings

### DISCUSSION THREAD

**Discussion by OpenSourceSimon** [plausible/analytics#3301] [Self-Hosted Support]
I'm following the [documentation ](https://plausible.io/docs/custom-props/props-dashboard) and I've just set up the custom properties analysis. Unfortunately I don't have a menu item for the custom properties, while the docs show that there needs to be one. Am I missing something?
![image](https://github.com/plausible/analytics/assets/65854503/30f2dbe6-4be5-4080-8fa4-23a7c2d4a9d2)

**Comment by pau1phi11ips** [comment]
https://plausible.io/docs/custom-props/introduction states this

> Custom properties is an upcoming premium feature that's free-to-use during the private preview. Pricing will be announced soon

**Reply by OpenSourceSimon** [reply]
How can I access the private preview?

**Reply by pau1phi11ips** [reply]
Only available using the plausible.io hosted scripts AFAIK. Custom props are not available on self-hosted installs 🫤

**Reply by OpenSourceSimon** [reply]
Very weird for a company that says they're completely open-source...
> We’re open source and all our code is available on GitHub.  ([Source](https://plausible.io/open-source-website-analytics))

**Reply by pau1phi11ips** [reply]
I'm a bit bummed about it too but they still have to pay the bills.

**Reply by OpenSourceSimon** [reply]
> I'm a bit bummed about it too but they still have to pay the bills.

Yeah of course, I totally get that. But then just don't call your project completely open-source.

**Reply by Daniel15** [reply]
Custom properties seem to work somewhat fine on the self-hosted version? There's probably some features missing, and the setting section doesn't appear, but filtering by them in Filter -> Property works:
![image](https://github.com/plausible/analytics/assets/91933/9fe582d2-eab9-4801-b2f3-216ed353b4ff)

And they also appear for custom events in the table of Goal Conversions too:
![image](https://github.com/plausible/analytics/assets/91933/108a9fd2-1003-454f-89a9-0309fb6ab529)

**Comment by fishbone1** [comment]
There is no hint in the docs that this is unavailable feature in the self-hosted version. Maybe the quote above has been removed meanwhile. It would be pretty annoying if true, now that we installed everything and customized our web site code to send those custom events.

**Reply by ruslandoga** [reply]
👋 @fishbone1 

[A release candidate for v2.1.0](https://github.com/plausible/analytics/discussions/3817) (which includes custom properties) is now available! :)
